packages/base/src/StaticAreaItem.js

```diff
@@ -37,9 +38,9 @@ class StaticAreaItem extends HTMLElement {
 	 */
 	update() {
 		if (this._rendered) {
-			this.setAttribute(this.pureTagName, "");
 			this._updateContentDensity();
 			this._updateDirection();
+			this._updateAdditionalAttrs();
```

Comment:
Maybe this is not a problem at all, but it would be safest IMO to move this method call to where the old `this.setAttribute` was (before updateContentDensity and updateDirection) so that we don't break any functionality in these methods that relies on `this.pureTagName`, if such exists.

Reply:
Agree 👍